conjure-java-core/src/integrationInput/java/com/palantir/product/AsyncRequestProcessingTestServiceEndpoints.java

```diff
@@ -96,6 +100,66 @@ public HttpHandler handler() {
         }
     }
 
+    private static final class DelayFiveSecondTimeoutEndpoint
+            implements HttpHandler, Endpoint, ReturnValueWriter<String> {
+        private static final ImmutableSet<String> TAGS = ImmutableSet.of("server-async{timeout=5 seconds}");
+
+        private final UndertowRuntime runtime;
+
+        private final UndertowAsyncRequestProcessingTestService delegate;
+
+        private final Serializer<String> serializer;
+
+        DelayFiveSecondTimeoutEndpoint(UndertowRuntime runtime, UndertowAsyncRequestProcessingTestService delegate) {
+            this.runtime = runtime;
+            this.delegate = delegate;
+            this.serializer = runtime.bodySerDe().serializer(new TypeMarker<String>() {});
+        }
+
+        @Override
+        public Set<String> tags() {
+            return TAGS;
+        }
+
+        @Override
+        public void handleRequest(HttpServerExchange exchange) throws IOException {
+            Map<String, Deque<String>> queryParams = exchange.getQueryParameters();
+            OptionalInt delayMillis = runtime.plainSerDe().deserializeOptionalInteger(queryParams.get("delayMillis"));
+            ListenableFuture<String> result = delegate.delayFiveSecondTimeout(delayMillis);
+            runtime.async().register(result, this, Duration.ofMillis(/* 5 seconds */ 5000), exchange);
```

Comment:
Example generated code. I retained the original human readable piece in a comment so that we can avoid expensive string parsing, while still producing readable source.